src/main/java/frc/robot/subsystems/swerve/Module.java

```diff
@@ -66,8 +67,12 @@ public void periodic() {
     odometryPositions = new SwerveModulePosition[sampleCount];
     for (int i = 0; i < sampleCount; i++) {
       double positionMeters = inputs.odometryDrivePositionsMeters[i];
-      Rotation2d angle = inputs.odometryTurnPositions[i];
-      odometryPositions[i] = new SwerveModulePosition(positionMeters - lastPositionMeters, angle);
+      NullableRotation2d angle = inputs.odometryTurnPositions[i];
+      if (angle.get() == null) {
+        odometryPositions[i] = null; // SwerveSubsystem deals with this
```

Comment:
Where does it deal with this? I see it doing a check on odometryYawPositions but not odometryPositions.

Reply:
https://github.com/HighlanderRobotics/Crescendo/blob/gyro-odo-fix/src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java#L388-L391